test/test-scheme

```diff
@@ -1,14 +1,6 @@
 #!/bin/bash
```

Comment:
Consider using [`#!/usr/bin/env bash`](https://unix.stackexchange.com/a/206366/240021) instead. I was actually burned by this on Shawn's machine after he upgraded bash 3.2 -> 5.0 when we were trying to get the `assets` script working.

Reply:
Was Shawn on a Mac? You can't update /bin there and must put the newer version into /usr/local/bin. I'll make this change.

Reply:
Yes, that's exactly what the problem was.

Reply:
Since I'm not using any bashisms, I'd prefer to use /bin/sh. That should work on all posix systems.